messagingpush/src/main/java/io/customer/messagingpush/data/model/CustomerIOParsedPushPayload.kt

```diff
@@ -0,0 +1,52 @@
+package io.customer.messagingpush.data.model
+
+import android.os.Bundle
+import android.os.Parcel
+import android.os.Parcelable
+
+/**
+ * Data class that holds payload for notifications from CIO dashboard
+ *
+ * @property extras data received in notification payload
+ * @property deepLink url received in notification data
+ * @property cioDeliveryId Customer.io message delivery id
+ * @property title notification content title text
+ * @property body notification content body text
+ */
+data class CustomerIOParsedPushPayload(
+    val extras: Bundle,
+    val deepLink: String?,
+    val cioDeliveryId: String,
+    val title: String,
+    val body: String
+) : Parcelable {
+    constructor(parcel: Parcel) : this(
+        extras = parcel.readBundle(Bundle::class.java.classLoader) ?: Bundle(),
+        deepLink = parcel.readString(),
```

Comment:
this can also be empty, can it not?

Reply:
This can, since it is `nullable`, we don't need to add fallback empty value to this